apps/acrs/behaviours/resume-form-session.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable consistent-return */
+'use strict';
+
+const axios = require('axios');
+const config = require('../../../config');
+const moment = require('moment');
+const baseUrl = `${config.saveService.host}:${config.saveService.port}/saved_applications/email/`;
+const _ = require('lodash');
+
+const encodeEmail = email => Buffer.from(email).toString('hex');
+
+const SESSION_DEFAULTS = config.sessionDefaults;
+
+module.exports = superclass => class extends superclass {
+  cleanSession(req) {
+    let cleanList = Object.keys(req.sessionModel.attributes);
+    const leaveFields = SESSION_DEFAULTS.fields.concat(['csrf-secret', 'user-cases']);
+    cleanList = cleanList.filter(item => leaveFields.indexOf(item) === -1);
+
+    req.sessionModel.unset(cleanList);
+    req.sessionModel.set('steps', SESSION_DEFAULTS.steps);
+  }
+  // GET lifecycle
+  configure(req, res, next) {
+    // reset session but keeping login fields, csrf and preexisting cases garnered from API
+    this.cleanSession(req);
+    // skip requesting data service api when running in local and test mode
+    if (config.env === 'local' || config.env === 'test') {
+      return super.configure(req, res, next);
+    }
+    return this.getCases(req, res, next);
+  }
+
+  getCases(req, res, next) {
+    const email = req.sessionModel.get('user-email');
+    const idType = req.sessionModel.get('id-type');
+
+    return axios.get(baseUrl + encodeEmail(email))
+      .then(response => {
+        const sessionCases = req.sessionModel.get('user-cases') || [];
+        const unsubmittedCases = _.filter(response.data, record => !record.submitted_at &&
+                                                                    record.session['id-type'] === idType);
```

Comment:
This is outdated code now I think. Might be worth merging your ACRS-61 branch to get the latest version of this file.